crates/red_knot_python_semantic/resources/mdtest/stubs/ellipsis.md

```diff
@@ -0,0 +1,42 @@
+# Ellipsis
+
+## Function and Methods
+
+For default values the ellipsis literal `...` can be used.
+
+```py
```

Comment:
This special case for `...` default values is only allowed in stub files, and the default mdtest path is `test.py`, which is not a stub file. So every test showing the special case working should have an explicit file path with `pyi` prefix, and we should also have a test for a `.py` filename showing that the special case is not applied, and a diagnostic for invalid default-value type is emitted.
```suggestion
For default values the ellipsis literal `...` can be used, in a stub file only.

```py path=test.pyi
```